CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* feat: `Felt252` now uses _lambdaworks_' `FieldElement` internally
```

Comment:
AFAICT it's a breaking change. We removed a `trait` among several other things.

Reply:
Added the changes to the changelog. We only removed `modpow` and `to_bytes_be`. The `FeltOps` trait we removed was `pub(crate)` 🗿